content/en/docs/ops/deployment/performance-and-scalability/index.md

```diff
@@ -104,7 +104,7 @@ average and tail latencies. The actual tail latency depends on the traffic patte
 ### Latency for Istio 1.21
 
 Inside the mesh, a request traverses the client-side proxy and then the server-side
-proxy. In the default configuration of Istio {{< istio_release_name >}} (i.e. Istio with telemetry v2),
+proxy. In the default configuration of Istio 1.21 (i.e. Istio with telemetry v2),
```

Comment:
why do we need 1.21 here? the page is not yet updated for 1.22 ?

Reply:
yes the page is not yet updated to 1.22, it should be updated after 1.22.1 release